core/conversion/converters/impl/stack.cpp

```diff
@@ -43,10 +43,9 @@ auto stack_registrations TORCHTRT_UNUSED = RegisterNodeConversionPatterns().patt
            auto cont = t.toCustomClass<TensorContainer>();
            itensor = cont->tensor();
          }
-
          auto shuffle_layer = ctx->net->addShuffle(*itensor);
          TORCHTRT_CHECK(shuffle_layer, "Unable to create shuffle layer from node: " << *n);
-         shuffle_layer->setReshapeDimensions(util::unsqueezeDims(itensor->getDimensions(), dim));
+         shuffle_layer->setReshapeDimensions(util::unsqueezeDims(itensor->getDimensions(), dim, 1, false));
```

Comment:
In the description of this PR, it says `Unsqueezed dims used 0 for dynamic dimension which was not aligned correctly with the dynamic dim in the input, switch to using -1.`. Here the `val` is 1 (which is typical of an unsqueeze op). Can you elaborate on what's going on here related to dynamic shapes?

Reply:
I'm referring to the last arg to unsqueezeDims `use_zeros` which previously defaulted to true and converted dynamic dimensions to 0 rather than -1.